src/Workleap.DomainEventPropagation.Subscription/ServiceCollectionEventPropagationExtensions.cs

```diff
@@ -13,4 +15,26 @@ public static IEventPropagationSubscriberBuilder AddEventPropagationSubscriber(t
 
         return new EventPropagationSubscriberBuilder(services);
     }
+
+    public static IEventPropagationSubscriberBuilder AddRateLimiting(this IEventPropagationSubscriberBuilder builder, Action<EventPropagationThrottlingOptions>? configure = null)
```

Comment:
Maybe you can add a more generic overload like the following to give more flexibility if needed

````c#
public static IEventPropagationSubscriberBuilder AddRateLimiting(this IEventPropagationSubscriberBuilder builder, RateLimiter limiter)
````